ballerina/errors.bal

```diff
@@ -17,6 +17,9 @@
 # Defines the common error type for the module.
 public type Error distinct error;
 
+# Error type to capture the errors occurred while retrieving messages in Azure service bus listener.
+public type MessageRetrievalError distinct (Error & error<ErrorContext>);
```

Comment:
Do we need to have parenthesis to group intersection subtypes? We don't do that for union types right?

```suggestion
public type MessageRetrievalError distinct (Error & error<ErrorContext>);
```

Reply:
Fixed with [7c4fc421dbda06e1870ff19e3c7d772fefc4f901](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/220/commits/7c4fc421dbda06e1870ff19e3c7d772fefc4f901)